images/disks-images-provider/entrypoint.sh

```diff
@@ -38,7 +39,12 @@ rm /images/fedora-cloud/disk.qcow2
 echo "copy all images to host mount directory"
 cp -R /images/* /hostImages/
 chmod -R 777 /hostImages
-chcon -Rt svirt_sandbox_file_t /hostImages
+
+# When the host is ubuntu, by default, selinux is not used, so chcon is not necessary.
+# If selinux tag is set, use chcon to change /hostImages privileges.
+if [ ${SELINUX_TAG:0:1} != "?" ]; then
```

Comment:
Nice, 
nit
do we want to also check  /etc/sysconfig/selinux or sestatus to see if its disabled, then to not even check the TAG?
just nice to have, in case you think its add value

beside that lgtm

Reply:
> Nice,
> nit:
> do we want to also check /etc/sysconfig/selinux or sestatus to see if its disabled, then to not even check the TAG?
> just nice to have, in case you think it adds value.
> 
> beside that lgtm
> Thanks

I tired to find a easy way to get selinux status. Unfortunitily, /etc/sysconfig/selinux file is not exist in the docker image and sestatus is not work either. 

This is the only way I found to verify selinux status.

Reply:
ye wasnt sure it works even, this is why i said "if it adds value"

Thanks for checking and the work

/lgtm